### DISCUSSION TITLE
Allow comments on individual commits in a PR

### DISCUSSION THREAD

**Discussion by wesbland** [community/community#5748] [Mobile]
My team values the fact that individual commits can be read on their own and the commit message can provide context about the change. Then all commits in a topic are grouped into a branch and a PR is created. As far as I can tell, there isn't a way to make comments on individual commits in the iOS app.

**Comment by alcere** [comment]
Hi @wesbland.
You are correct, right now the apps are not displaying comments on code that belongs to individual commits, you need to go to the pull request review screen to see those. I'm going to include your feature request to our internal tracking issue.

**Reply by wesbland** [reply]
Just so it's clear, I want to able read **and write** comments on PR commits.

**Reply by dmnks** [reply]
Isn't the request about being able to comment on individual commits themselves, such as on the commit messages? If so, that would be a nice feature, indeed! (And not just for mobile, but generally.)

**Reply by wesbland** [reply]
I originally intended to be able to write the response on the code when viewing an individual commit, but being able to comment on a commit message would be nice too. I often want to do that even when doing a review in the browser (where that's also not possible).
